src/main/java/org/jenkinsci/plugins/mesos/MesosRetentionStrategy.java

```diff
@@ -0,0 +1,9 @@
+package org.jenkinsci.plugins.mesos;
+
+import hudson.slaves.CloudRetentionStrategy;
+
+public class MesosRetentionStrategy extends CloudRetentionStrategy {
```

Comment:
Should this be passed to the `MesosSlave`? What is the effect of the retention strategy?